botx/models/enums.py

```diff
@@ -116,6 +116,9 @@ class MentionTypes(str, Enum):
     #: mention channel in message.
     channel = "channel"
 
+    #: mention each members
```

Comment:
mention each member

Reply:
synonymous description is better way in comments

Reply:
"Each" hears as iteration, "all" as one operation.